dask/array/rechunk.py

```diff
@@ -242,19 +193,37 @@ def rechunk(x, chunks, threshold=DEFAULT_THRESHOLD,
     block_size_limit: int
         The maximum block size (in bytes) we want to produce during an
         intermediate step.
```

Comment:
No longer an "intermediate step", but actual end chunk size

Reply:
I've removed the intermediate step text